android/src/main/java/com/swmansion/rnscreens/utils/EdgeToEdge.kt

```diff
@@ -0,0 +1,13 @@
+package com.swmansion.rnscreens.utils
+
+// https://github.com/zoontek/react-native-edge-to-edge/blob/main/react-native-is-edge-to-edge/README.md
+object EdgeToEdge {
```

Comment:
```suggestion
object EdgeToEdgePackageDetector {
```

Let's give it some more meaningful name.

Reply:
0757fcbe9658a9d3c6aeaf39b5db14a96a406c7b